salesagility/SuiteCRM
# Log level "Warn" cleanup

## Description



This fixes the suitecrm.log from getting filled with spam when set to log level "Warn". This should solve all warning mentioned by @pgorod in #3401. In total:

- Image List.gif not found
- Image Create.gif not found
- Image View_Email_Templates.gif not found
- Image Schedule_Meeting.gif not found
- Image Schedule_Call.gif not found
- Image Resource_Chart.gif not found
- Image View_Project_Tasks.gif not found
- Image Create_Person_Form.gif not found
- Image View_Create_Email_Templates.gif not found
- Image View_Diagnostics.gif not found
- Image Create_Security_Group.gif not found
- Image Create_Contact_Vcard.gif not found
- Image View.gif not found
- Image View_Process_Audit.gif not found
- Image Today.gif not found
- Image CREATEOutboundEmailAccounts.gif not found
- Image EmailMarketing.gif not found
- Image Role_Management.gif not found
- Image Quick_Radius_Map.gif not found
- Image Create_Lead_Vcard.gif not found
- Image List_Maps.gif not found
- Image icon_Administration_32.png not found
- Image icon_EmailMan_32.png not found
- Image icon_Home_32.png not found
- Image icon_UpgradeWizard_32.png not found
- Image icon_InboundEmail_32.png not found
- Image _image_inline.gif not found
- Image png_image_inline.gif not found
- Image icon__32.png not found
- Image sub_panel\EmailMarketing.svg not found
- Image icon_loading.gif not found
- Image sidebar/modules\InboundEmail.svg not found
- Image select.gif not found
- Image clear.gif not found

## Motivation and Context


Related to #3398 and #3401

## How To Test This

1. Set log level to "Warn".
2. Check that the log is not spammed with "Image not Found".

## Types of changes


### Final checklist
